app/logic/events.py

```diff
@@ -64,8 +64,11 @@ def attemptSaveEvent(eventData, attachmentFiles = None):
         return False, validationErrorMessage
 
     try:
-        events = saveEventToDb(newEventData)
-        if attachmentFiles:
+        if  attachmentFiles == None:
+            saveEventToDb(newEventData)
+            return True, ""
+        else:
+            events = saveEventToDb(newEventData)
```

Comment:
Let's reorganize this so that the attachmentFiles check just decides whether we do the addfile loop. Because everything else is the same